src/rpc/rate-limit.js

```diff
@@ -0,0 +1,83 @@
+/*
+  Rate limit
+*/
+/* eslint no-useless-catch: 0 */
+
+// Local libraries
+const RateLimitLib = require('koa2-ratelimit').RateLimit
+
+class RateLimit {
+  constructor (options) {
+    // Encapsulate dependencies
+    this.RateLimitLib = RateLimitLib
+
+    // Set default rate limit options.
+    this.defaultOptions = {
+      interval: { min: 1 },
+      max: 60,
```

Comment:
I set the default to 60 requests per minute. This impacted the unit test, but I don't think it should have.